packages/adblocker/src/resources.ts

```diff
@@ -307,7 +326,9 @@ export default class Resources {
       dataUrl = `data:${contentType};base64,${btoaPolyfill(body)}`;
     }
 
-    return { body, contentType, dataUrl };
+    // TODO: Direct response from `@remusao/small`
```

Comment:
Can this be resolved now that the referenced PR has been merged and released?

Reply:
updated in ff1fe73f0ac2019ecc2c64564ba22df00b297ed5